frontend/eda/common/TeamAccess.tsx

```diff
@@ -0,0 +1,185 @@
+import { useTranslation } from 'react-i18next';
+import {
+  compareStrings,
+  IPageAction,
+  ITableColumn,
+  PageActionSelection,
+  PageActionType,
+  PageTable,
+  IToolbarFilter,
+  ToolbarFilterType,
+} from '../../../framework';
+import { edaAPI } from './eda-utils';
+import { useCallback, useMemo } from 'react';
+import { ButtonVariant } from '@patternfly/react-core';
+import { MinusCircleIcon } from '@patternfly/react-icons';
+import { useEdaView } from './useEventDrivenView';
+import { useEdaBulkConfirmation } from './useEdaBulkConfirmation';
+import { idKeyFn } from '../../common/utils/nameKeyFn';
+import { requestDelete } from '../../common/crud/Data';
+
+export type TeamAssignment = {
```

Comment:
Can we create a directory `frontend/eda/access/interfaces` and move this type as a file under there?  It'll make it easy to locate access related interfaces/types.